Dockerfile

```diff
@@ -20,7 +20,7 @@ FROM confluentinc/cp-enterprise-replicator:$CP_VERSION
 ARG CONNECTOR_VERSION
 
 ENV CONNECT_PLUGIN_PATH: "/usr/share/java,/usr/share/confluent-hub-components"
-
+ENV CLASSPATH: "/usr/share/java/kafka-connect-replicator/replicator-rest-extension-$CP_VERSION.jar:/usr/share/java/monitoring-interceptors/monitoring-interceptors-$CP_VERSION.jar"
```

Comment:
@bbejeck 

1. I may be misinterpreting what https://confluentinc.atlassian.net/browse/REPL-1224 resolved , but I thought it removed the need to set the `CLASSPATH` at all (@thomaskwscott )
2. Even so, why is `CLASSPATH` required in the Dockerfile (versus just the `docker-compose.yml` file)

Reply:
I wasn't sure about this. I was about to take this out, but I wanted to leave it in the review for confirmation.  I'll remove now